server/index.js

```diff
@@ -258,12 +262,90 @@ app.use('/api/apps', async (req, res) => {
 
 app.use('/api/images', express.static('./server/assets'))
 
+// ------------------------------------------------
+// ---------- Pages with Open Graph tags ----------
+// ------------------------------------------------
+
+app.use('/apps', (req, res) => {
+    res.render('index', {
+        'title': 'Apps',
+        'path': req.originalUrl,
+    })
+})
+
+app.use('/add', (req, res) => {
+    res.render('index', {
+        'title': 'Add Feed',
+        'path': req.originalUrl,
+    })
+})
+
+
+app.use('/podcast/value4value', (req, res) => {
+    res.render('index', {
+        'title': 'Value 4 Value Podcasts',
+        'description': 'These podcasts are set up to receive Bitcoin payments in real-time over the Lightning network using compatible Podcasting 2.0 apps.',
+        'path': req.originalUrl,
+    })
+})
+
+app.use('/podcast/:podcastid(\\d+)', async (req, res) => {
+    const params = {
+        'title': 'Podcast', // default title
+        'path': req.originalUrl,
+    }
+
+    if (req.query.episode) { // /podcast/41504?episode=15576217793
+        const response = await api.episodesById(req.query.episode)
+
+        if (response?.episode?.id) {
+            const { feedTitle, title, description, image } = response.episode
+            const textDescription = description.replaceAll(/<\/?[^>]+(>|$)/gi, '').replaceAll(/[\r\n]/g, ' ')
```

Comment:
If we keep running on node 14, `replaceAll` should be changed to `replace` since it isn't supported.

See https://stackoverflow.com/a/66937599

> Nodejs v14 has V8 8.4 But, replaceAll was added to V8 8.5.

Reply:
I think I was running Node 16 or 18 when developing this code. Replacing it with `replace` should be fine or with anything that can convert the HTML description into a plain-text description.

Reply:
I started on 20 and worked back through 18 and 16 before finding the issue in 14. When I switched it with `replace`, I found it worked the same as before.